### PR TITLE
WIP

### PR DESCRIPTION
**Pull request by metmarkosaric**

### CODE REVIEW

File: docs/custom-event-goals.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Tag the site element that you want to track with a CSS class name. How to do thi
 
 For instance, if you're using WordPress, you can click on any block element you want to track such as a button. This will open up the block menu on the right-hand side of your screen. You can click on "Advanced" and add a CSS class name in the "Additional CSS class(es)" field.
 
-Add the CSS class name in this format: `plausible-event-name=custom-event`. For instance, if you want to track form submissions on your contact form, you could use: `plausible-event-name=form-submit`.
+Add the CSS class name in this format: `plausible-event-name=MyEventName`. For instance, if you want to track form submissions on your contact form, you could use: `plausible-event-name=Form+Submit`.
```
Comment (RobertJoonas) [inline docs/custom-event-goals.md:37]: I thought the `custom-event` there might be misleading. `MyEventName` makes it clearer what the meaning of this value is

Comment (RobertJoonas) [reply docs/custom-event-goals.md:37]: About the `Form+Submit` part, I thought it's better to advertise uppercase words with spaces for event names. Would be consistent with `Outbound Link: Click` and `File Download`

Comment (RobertJoonas) [reply docs/custom-event-goals.md:37]: Yes, that's right. I added a note about it in the parentheses. Also, there's a note that you can use `+` to represent a space, so it's mentioned in two places already. Any ideas on how to make it even clearer?

Comment (metmarkosaric) [reply docs/custom-event-goals.md:37]: cool, thanks! one mention is for custom props and i didn't want to mention custom props in the top part before we actually introduce what they are. I'll review this a few more times to make more changes.

Comment (RobertJoonas) [reply docs/custom-event-goals.md:37]: > i didn't want to mention custom props in the top part before we actually introduce what they are

That's a very good point. Makes it much cleaner.

Perhaps rephrasing the "(plus is replaced by a space in the end)" with a bit longer explanation would make it more clear. I'll leave it up to you if that's OK.

**Pull request merged**